tests/integration/Storefront/Page/Account/AccountOrderPageLoaderTest.php

```diff
@@ -99,6 +100,38 @@ public function testLogsInGuestById(): void
         );
     }
 
+    public function testLoad(): void
+    {
+        $salesChannel = $this->createSalesChannelContextWithLoggedInCustomerAndWithNavigation();
+
+        $orderId = $this->placeRandomOrder($salesChannel);
+        /** @var OrderEntity $order */
```

Comment:
please do not use var annotations. Especially in tests you can simply assert the type. also see https://github.com/shopware/shopware/blob/trunk/coding-guidelines/core/writing-code-for-static-analysis.md